tools/docker/tests/test_flows.sh

```diff
@@ -636,19 +641,19 @@ main() {
 
     if [ $error -gt 0 ]; then
         err "$error / $count tests failed"
-        if [ "$VERBOSE" = "true" ]; then
-            for dockerimage in gateway repeater1 repeater2; do
-                info "*** Dumping logs from $dockerimage ***"
-                docker exec $dockerimage sh -c '
-                    for logfile in controller agent agent_wlan0 agent_wlan2; do
-                        logpath=/tmp/$USER/beerocks/logs/beerocks_$logfile.log
-                        [ -e $logpath ] && {
-                            printf "\n\n==> %s <==\n" $logfile
-                            cat $logpath
-                        }
-                    done'
-            done
-        fi
+        # if [ "$VERBOSE" = "true" ]; then
+        #     for dockerimage in gateway repeater1 repeater2; do
+        #         info "*** Dumping logs from $dockerimage ***"
+        #         docker exec $dockerimage sh -c '
+        #             for logfile in controller agent agent_wlan0 agent_wlan2; do
+        #                 logpath=/tmp/$USER/beerocks/logs/beerocks_$logfile.log
+        #                 [ -e $logpath ] && {
+        #                     printf "\n\n==> %s <==\n" $logfile
+        #                     cat $logpath
+        #                 }
+        #             done'
+        #     done
+        # fi
```

Comment:
Not related to this PR. But indeed a separate PR should be opened to remove this stuff, it was only relevant for Travis.